cmd/kubectl-gadget/deploy.go

```diff
@@ -128,6 +131,21 @@ func init() {
 		"debug", "d",
 		false,
 		"show extra debug information")
+	deployCmd.PersistentFlags().StringVar(
+		&socketPathContainerd,
+		"socketpath-containerd",
+		"",
+		"socket path to use for containerd runtime")
+	deployCmd.PersistentFlags().StringVar(
+		&socketPathCRIO,
+		"socketpath-crio",
+		"",
+		"socket path to use for CRI-O runtime")
+	deployCmd.PersistentFlags().StringVar(
+		&socketPathDocker,
+		"socketpath-docker",
+		"",
+		"socket path to use for docker runtime")
```

Comment:
IMO, we should align the naming notation with what local-gadget is doing. Maybe `socket-<runtime>` is a better naming:

https://github.com/kinvolk/inspektor-gadget/blob/5b93ab18b3b22741b26ae326d259156928d0cd54/cmd/local-gadget/utils/flags.go#L134-L153

Reply:
Yes, definitely they should be the same.
But like you asked: Which one should we pick :D `<runtime>-socket` or `socket-<runtime>`

Reply:
Mmmm I am ok either way. Pick the one you prefer! 🙂